app/labor/rate_limit_checker.rb

```diff
@@ -1,6 +1,10 @@
 class RateLimitChecker
   attr_reader :user, :action
 
+  def self.daily_account_follow_limit
+    ENV["DAILY_ACCOUNT_FOLLOW_LIMIT"]&.to_i || 500
```

Comment:
this should go inside the Envfile and be replaced with `ApplicationConfig["DAILY_ACCOUNT_FOLLOW_LIMIT"]`, maybe down here: 

https://github.com/thepracticaldev/dev.to/blob/b6a731276e260a009fd7f048586cf12f55599604/Envfile#L10-L15

I would also rename it to something like `RATE_LIMIT_FOLLOW_COUNT_DAILY`  so that if there are other variables in the future related to rate limiting they can be called `RATE_LIMIT_xyz` and be found easily :D